BlockchainSdk/Extensions/Task+.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Task+.swift
+//  BlockchainSdk
+//
+//  Created by Dmitry Fedorov on 08.04.2024.
+//
+
+import Foundation
+
+extension Task where Failure == Error {
+    @discardableResult
+    static func retrying(
+        priority: TaskPriority? = nil,
+        maxRetryCount: Int = 1,
+        retryDelay: TimeInterval = 0,
+        operation: @Sendable @escaping () async throws -> Success
+    ) -> Task {
+        Task(priority: priority) {
+            for _ in 0..<maxRetryCount {
+                do {
+                    return try await operation()
+                } catch {
+                    let oneSecond = TimeInterval(1_000_000_000)
+                    let delay = UInt64(oneSecond * retryDelay)
+                    try await Task<Never, Never>.sleep(nanoseconds: delay)
```

Comment:
А можно же так, или тут доп смысл есть?
```suggestion
                    try await Task.sleep(nanoseconds: delay)
```

Reply:
тут компилятор ругается:
Referencing static method 'sleep(nanoseconds:)' on 'Task' requires the types 'Success' and 'Never' be equivalent